tests/queries/0_stateless/00604_show_create_database.sql

```diff
@@ -1,5 +1,5 @@
--- Tags: no-ordinary-database, no-parallel
+-- Tags: no-ordinary-database
 
-create database if not exists test_00604;
-show create database test_00604;
-drop database test_00604;
+create database if not exists {CLICKHOUSE_DATABASE:Identifier};
+show create database {CLICKHOUSE_DATABASE:Identifier};
+drop database {CLICKHOUSE_DATABASE:Identifier};
```

Comment:
Because we don't use the default database:
https://github.com/ClickHouse/ClickHouse/blob/fb903727c2f3a8dda746d35915ecdc6636a898e0/tests/clickhouse-test#L756
https://github.com/ClickHouse/ClickHouse/blob/fb903727c2f3a8dda746d35915ecdc6636a898e0/tests/clickhouse-test#L1305-L1306

So replacing `test_00604` and similar with `{CLICKHOUSE_DATABASE:Identifier}` makes tests use unique database names, so it's possible to run them in parallel

Reply:
Wow, I missed the normalization and that confused me. But you are totally right.